src/createServer.js

```diff
@@ -1,3 +1,62 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const { detectCase } = require('./convertToCase/detectCase');
+
+const supportedCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const urlParts = req.url.split('?');
+    const textToConvert = urlParts[0].slice(1);
+    const queryString = urlParts[1];
+    const params = new URLSearchParams(queryString);
+    const toCase = params.get('toCase');
+
+    const errors = [];
+
+    if (!textToConvert) {
+      errors.push({
+        message: 'Text to convert is required.'
+          + ' Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      });
+    }
+
+    if (!toCase) {
+      errors.push({
+        message: '"toCase" query param is required.'
+          + ' Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
```

Comment:
This part of message used in to message, you can create separate variable for this part